src/ert/job_queue/driver.py

```diff
@@ -52,7 +52,8 @@ def create_driver(cls, queue_config: QueueConfig) -> "Driver":
         driver = Driver(queue_config.queue_system)
         if queue_config.queue_system in queue_config.queue_options:
             for setting in queue_config.queue_options[queue_config.queue_system]:
-                driver.set_option(*setting)
+                if not driver.set_option(*setting):
+                    raise ValueError(f"Queue option not set {setting}")
```

Comment:
Can we in some way blow up something with this? 🤔

Reply:
That's the whole point I think at least, that those errors should be caught before we hit start simulation.

Reply:
Me thinking there are rogue ERT configs out in the wild. Hopefully the user will get warnings from the new validation part before we hit this. Maybe we are quite sure on that part. I assume it is impossible to reach this ValueError if we go through the validated config, and only if we have rogue self-written application utilizing the c-driver?

Reply:
I see your point point. If i get your point right, instead of raising an error we should provide a a warning? The problem with warning that this might get ignored :)